spec/lib/describe_indexer_spec.rb

```diff
@@ -81,5 +93,60 @@
         expect(indexed_record["uri_ssim"].include?("https://doi.org/10.34770/r75s-9j74")).to eq true
       end
     end
+
+    context "files" do
+      # TODO: Index filenames for PDC Describe objects
+      xit 'files' do
+        # The fixture has three files but we expect two of them to be ignored
+        files = JSON.parse(result['files_ss'].first)
+        expect(files.count).to eq 1
+        expect(files.first["name"]).to eq 'Lee_princeton_0181D_10086.pdf'
+      end
+    end
```

Comment:
This came from the duplicate (and now deleted) file `spec/lib/pdc_describe_indexer_spec.rb`